docs/01-engage/02-accounts/03-contact-details/01-enable-contact-preferences/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Enable Contact Preferences
+sidebar_position: 1
+---
+
+Contact is simply a way that involves meeting or communicating with anyone. So, when a contact wants to receive communication by any means, whether it is by telephone or email we call it a **contact preference**. This is important in terms of knowing and maintaining a valuable relationship with the donor or sponsor (the one you are contacting). 
+
+## Contact Preferences in ENGAGE
+
+ENGAGE allows you to prioritize your contact preferences while creating an account, be it individual or by an organization. There are in total four different types of preferences that ENGAGE provides:
+
+- Telephone
+- Email
+- SMS
+- Post
+
+Each preference contains options that clarify if a contact wants to be contacted for a specific purpose or not.
+
+### Unspecified
+
+This implies that the account owner does not know exactly what they want. For example, the donor might engage themselves in communication methods in the future.
```

Comment:
This I think take out, but it isn't right if being left.

It can be the account doesn't know what they want and said leave it unspecified, but it is much more likely they haven't been asked/answered in the first place.